packages/react-native/src/private/webapis/performance/specs/NativePerformanceObserver.js

```diff
@@ -54,6 +58,8 @@ export interface Spec extends TurboModule {
     entryName?: string,
   ) => $ReadOnlyArray<RawPerformanceEntry>;
   +getSupportedPerformanceEntryTypes: () => $ReadOnlyArray<RawPerformanceEntryType>;
+  
+  +getEventCounts: () => $ReadOnlyArray<[string, number]>;
```

Comment:
Nit: please keep it in the same position so it doesn't show up as a new thing in the PR.